kuma/static/js/highlight.js

```diff
@@ -0,0 +1,16 @@
+(function(win, doc, $) {
```

Comment:
Nit: As `win` and `doc` is not used inside the code block, it is safe to only pass `jQuery` an hence, receive only `$`. Currently is is probably triggering an `eslint` warning about `no-unused-vars`